libs/hbb_common/src/config.rs

```diff
@@ -77,12 +77,28 @@ const CHARS: &'static [char] = &[
     'm', 'n', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z',
 ];
 
+//check for env variable RENDEZVOUS_SERVER1-3 if not use the default
 pub const RENDEZVOUS_SERVERS: &'static [&'static str] = &[
-    "rs-ny.rustdesk.com",
-    "rs-sg.rustdesk.com",
-    "rs-cn.rustdesk.com",
+    match option_env!("RENDEZVOUS_SERVER1") {
+        Some(key) => key,
+        None => "rs-ny.rustdesk.com",
+    },
+    match option_env!("RENDEZVOUS_SERVER2") {
+        Some(key) => key,
+        None => "rs-sg.rustdesk.com",
+    },
+    match option_env!("RENDEZVOUS_SERVER3") {
+        Some(key) => key,
+        None => "rs-cn.rustdesk.com",
```

Comment:
If `RENDEZVOUS_SERVER2 ` is not set, still use rustdesk server? looks wierd.

Reply:
That is correct see above

Reply:
This item the issue I am running into is there is limited logic i can do in global scope, I cant figure out another way to do it... I am open to ideas but I have tried a bunch of things and rust just limits the global scope too much